File: src/dataset/typescript/raindrops_Exercism-typescript-shybyte/claude/ClaudeSonnet4_raindrops_v2.ts
export default class Raindrops {
    convert(n: number) {
        const div3 = n % 3 === 0;
        const div5 = n % 5 === 0;
        const div7 = n % 7 === 0;
        
        if (!(div3 || div5 || div7)) return n.toString();
        
        return (div3 ? 'Pling' : '') + 
               (div5 ? 'Plang' : '') + 
               (div7 ? 'Plong' : '');
    }
}

File: src/dataset/Java/raindrops_exercism-java-uzilan_robiworks/claude/ClaudeSonnet4_raindrops_v2.java
class RaindropConverter {

    String convert(int number) {
        boolean hasSound = false;
        StringBuilder output = new StringBuilder(15);
        
        if (number % 3 == 0) {
            output.append("Pling");
            hasSound = true;
        }
        if (number % 5 == 0) {
            output.append("Plang");
            hasSound = true;
        }
        if (number % 7 == 0) {
            output.append("Plong");
            hasSound = true;
        }
        
        return hasSound ? output.toString() : Integer.toString(number);
    }

}